ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/private/cucushift-installer-rehearse-azure-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-private
```

Comment:
From the PR title, I just realized this PR is for a fully-private cluster + behind proxy, I think the workflow name should reflect that, something like "ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/proxy-fullyprivate/cucushift-installer-rehearse-azure-ipi-proxy-fullyprivate-workflow.yaml"

Reply:
I dont think I can give a dash in the file name. If fully-private needs to be in the name, then the folder structure is going to be `ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/proxy/fully/private/cucushift-installer-rehearse-azure-ipi-proxy-fully-private-workflow.yaml`

Reply:
Hmm, I did not try yet, but I saw some existing dir name includes a dash in its name. To avoid confusion, I guess we can name it `fullyprivate`.